messaging/src/main/java/org/axonframework/eventhandling/ReplayToken.java

```diff
@@ -39,11 +39,11 @@
 public class ReplayToken implements TrackingToken, WrappedToken, Serializable {
 
     private static final long serialVersionUID = -4102464856247630944L;
-    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
+    @JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "@class")
```

Comment:
The used property isn't deprecated, but did cause flaky build warnings. As `JsonTypeInfo.As.PROPERTY` is the default for the `include` attribute any how, I removed it (thus resolving the warnings).